boards/openmote-b/board.c

```diff
@@ -36,9 +36,15 @@ void board_init(void)
     gpio_init(RF24_SWITCH_CC2538_PIN, GPIO_OUT);
     gpio_init(RF24_SWITCH_AT86RF215_PIN, GPIO_OUT);
 
-    /* start with cc2538 2.4ghz radio*/
+#ifdef MODULE_AT86RF215
+    /* use at86rf215 2.4ghz radio*/
+    RF24_SWITCH_CC2538_OFF;
+    RF24_SWITCH_AT86RF215_ON;
+#else
+    /* use cc2538 2.4ghz radio*/
     RF24_SWITCH_CC2538_ON;
     RF24_SWITCH_AT86RF215_OFF;
+#endif
```

Comment:
To allow running both radios.

```suggestion
#ifdef MODULE_AT86RF215
    RF24_SWITCH_AT86RF215_ON;
#else
    RF24_SWITCH_AT86RF215_OFF;
#endif
#ifdef MODULE_CC2538_RF
    RF24_SWITCH_CC2538_ON;
#else
   RF24_SWITCH_CC2538_OFF;
#endif
```

Reply:
Only one radio can be connected to the 2.4 GHz antenna at a time.
However, by turning the conditions around we can make sure that the 2.4 GHz antenna is always connected to `cc2538_rf` if the module is used.

The sub-GHz antenna is hardwired to `at86rf215`.

Reply:
Oh got it, thanks for the clarification!